proto/catalog.proto

```diff
@@ -231,6 +231,11 @@ message Table {
   optional uint64 initialized_at_epoch = 28;
   optional uint64 created_at_epoch = 29;
 
+  // This field is introduced in v1.2.0. It is used to indicate whether the table should use
+  // watermark_cache to avoid state cleaning as a performance optimization.
+  // In older versions we can just initialize without it.
+  optional bool use_watermark_cache = 30;
```

Comment:
Why `optional`? I guess the default `false` is ok?

Reply:
I see. https://protobuf.dev/programming-guides/proto3/#default